tests/app/shared/issue-tables/issue-sorter.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { getSortedData } from '../../../../src/app/shared/issue-tables/issue-sorter';
+import { MatSort } from '@angular/material';
+import { Issue } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_ASSIGNEES, ISSUE_WITH_EMPTY_DESCRIPTION, ISSUE_PENDING_MODERATION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+
+describe('issuer-sorter', () => {
+  describe('getSortedData()', () => {
+    const dummyTeam = new Team({
+      id: 'F09-2',
+      teamMembers: [],
+    });
+    const dummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    const otherDummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_ASSIGNEES, dummyTeam);
+    const issuesList: Issue[] = [dummyIssue, otherDummyIssue];
+
+    const dummyTodoIssue: Issue = Issue.createPhaseModerationIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    const otherDummTodoIssue: Issue = Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION, dummyTeam);
+    const todoIssuesList: Issue[] = [dummyTodoIssue, otherDummTodoIssue];
+    const matSort: MatSort = new MatSort();
+
+    it('sorts issues based on their assignees correctly', () => {
+      matSort.active = 'assignees';
+      matSort.direction = 'asc';
+      const sortedIssuesByTitleAsc = getSortedData(matSort, issuesList);
+      expect(sortedIssuesByTitleAsc[0].id).toBe(dummyIssue.id);
+      expect(sortedIssuesByTitleAsc[1].id).toBe(otherDummyIssue.id);
```

Comment:
To make the test more concise, should we have a method that verifies the order of the two issues?
```
expect(sortedIssuesByTitleAsc[0].id).toBe(dummyIssue.id);
expect(sortedIssuesByTitleAsc[1].id).toBe(otherDummyIssue.id);
```

could be replaced by `assertOrder(sortedIssuesByTitleAsc, dummyIssue, otherDummyIssue)`
?

Reply:
i see. as a result, we can just abstract out these checks in our helped method!

Is it possible that we do this for the tests in this `issue-sorter` ?

Reply:
Yea, the `assertOrder` helper will just be meant for `issue-sorter` - just to reduce the need for duplicating the assertions in all the tests.